### PR TITLE
Polishing Actually enable fat db pr (#1974)

### PR DESCRIPTION
**Pull request by debris**
finishes #1974 using #2014

Should be merged only if we decide to carry on with user defaults (#2014).

### CODE REVIEW

**Comment by gavofyork** [comment]
will need resolving

File: parity/cli.rs [hunk 1]
```diff
@@ -204,17 +204,24 @@ Sealing/Mining Options:
...
                            default to fast if none synced [default: auto].
+  --tracing BOOL           Indicates if full transaction tracing should be
+                           enabled. Works only if client had been fully synced
+                           with tracing enabled. BOOL may be one of auto, on,
+                           off. auto uses last used value of this option (off
+                           if it does not exist) [default: auto].
+  --fat-db BOOL            Build appropriate information to allow enumeration
+                           of all accounts and storage keys. Doubles the size
+                           of the state database. BOOL may be one of on,
+                           off. [default: off].
```
Comment (gavofyork) [inline parity/cli.rs]: should have an `auto`/`last` option now, too.

Comment (debris) [inline parity/cli.rs]: Done here: https://github.com/ethcore/parity/pull/2048/files#diff-a446157098e33897a7686a206229874aR223

**Comment by gavofyork** [comment]
conflicts...

**Comment by debris** [comment]
resolved :)

**Comment by gavofyork** [comment]
`net_connect` test is failing - unrelated?

**Comment by debris** [comment]
definitely

**Comment by debris** [comment]
I restarted travis job

**Pull request merged**